app-crypt/keybase/keybase-1.0.48.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils systemd user versionator
+
+MY_PV=$(replace_version_separator 3 '-')
+
+DESCRIPTION="Client for keybase.io"
+HOMEPAGE="https://keybase.io/"
+SRC_URI="https://github.com/keybase/client/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	>=dev-lang/go-1.6:0
```

Comment:
why do not use proper eclasses to handle go packages?

Reply:
Copy-pasted from the previous version, tell me how please.

Reply:
So far I haven't been able to get it to work with that eclass to be honest. I'll keep trying...

Reply:
I take that back @Zlogene. I just pushed an updated PR which is using the `golang-build` eclass.